guacamole/src/main/webapp/app/navigation/directives/guacUserMenu.js

```diff
@@ -95,13 +95,10 @@ angular.module('navigation').directive('guacUserMenu', [function guacUserMenu()
              */
             $scope.role = null;
 
-            // Pull user data
+            // Display user profile attributes if available
             userService.getUser(authenticationService.getDataSource(), $scope.username)
                     .then(function userRetrieved(user) {
 
-                // Store retrieved user object
-                $scope.user = user;
```

Comment:
Just want to make sure this doesn't have any impact?  Was this $scope.user not used at all?

Reply:
Ping @mike-jumper - just wanted to verify this and then I think I can push forward with merging this PR.